source/data_analysis_SPRING2023.Rmd

```diff
@@ -1,229 +1,1296 @@
+
+De hoge waarden voor collineariteit (Fig. \@ref(fig:check2)) zijn in dit geval geen probleem en zijn het gevolg van (verwachte) collineariteit t.g.v. toevoegen van interacties.
+
+```{r check2, fig.height=12, fig.cap = "Visuele controle van verschillende modelaannames."}
+performance::check_model(model1)
+```
+
+```{r, fig.cap = "Verwacht aantal soorten per soortgroep en methode op basis van ons model."}
+marginaleffects::plot_predictions(
+  model1,
+  condition = c("method_combi", "taxgroup"),
+  type = "response",
+  vcov = TRUE,
+  draw = FALSE) %>%
+  left_join(
+    data_sp_rich %>%
+      distinct(method_cd, spring_code, uv, level, method_combi, taxgroup)
+  ) %>%
+  as_tibble() %>%
+  ggplot() +
+  geom_pointrange(
+    aes(
+      x = method_cd, y = estimate, ymin = conf.low, ymax = conf.high,
+      colour = method_combi),
+    position = position_dodge(width = 0.5)
+  ) +
+  labs(y = "Soortenrijkdom") +
+  facet_wrap(~taxgroup, scales = "free")
+```
+
+## Alpha biodiversity
+
+De Shannon index wordt als volgt berekend:
+$$H' = - \sum_{i=1}^{R} p_i ln p_i$$
+Waarbij $p_i$ de proportie is van het aantal individuen van één soort ten opzichte van het totaal aantal individuen.
+
+Er kan geen Shannon index berekent worden voor pan traps of deeltransecten waarin niks gevangen werd. Het gebruik van deze index leidt dus sowieso tot een verlies aan informatie. Daarnaast is de Shannon index van een sample waar slechts één soort gevonden wordt gelijk aan 0.
+
+```{r}
+sha <- apoidea %>%
+  mutate(taxgroup = "Apoidea") %>%
+  bind_rows(
+    syrphidae %>%
+      mutate(taxgroup = "Syrphidae")
+  ) %>% 
+  group_by(sampling_site_cd, spring_code, method_cd, taxgroup,
+           location_code, method_combi) %>% 
+  mutate(tot_ind = sum(no_ind)) %>% 
+  group_by(sampling_site_cd, spring_code, 
+           method_cd, species_nm, tot_ind,
+           taxgroup,
+           location_code, method_combi) %>% 
+  summarise(n_ind = sum(no_ind)) %>% 
+  mutate(prop = (n_ind/tot_ind) * log(n_ind/tot_ind)) %>% 
+  group_by(sampling_site_cd, spring_code, 
+           method_cd, taxgroup,
+           location_code, method_combi) %>% 
+  summarise(exp_sh = exp(-sum(prop, na.rm = TRUE)))
+```
+
+```{r, fig.cap = "Shannon index (exp) per pan trap of subtransect van 50m."}
+ggplot(sha, aes(x = exp_sh)) +
+  geom_histogram(
+    aes(fill = taxgroup),
+    position = position_dodge())
+```
+
+```{r}
+model0 <- glmmTMB(
+  exp_sh ~ 
+    (method_combi +
+       location_code) * taxgroup
+  ,
+  family = "Gamma",
+  ziformula = ~ taxgroup,
+  na.action = na.fail,
+  data = sha
+)
+```
+
+```{r check3, fig.height=10, fig.cap = "Visuele controle van verschillende modelaannames."}
+performance::check_model(model0)
+```
+
+```{r, fig.cap = "Verwachte Shannon index (exp) per soortgroep en methode op basis van ons model."}
+marginaleffects::plot_predictions(
+  model0,
+  condition = c("method_combi",  "taxgroup"),
+  type = "response",
+  vcov = TRUE
+)
+```
+
+# Kosten
```

Comment:
Kan je in dit kostenluik ook zorgen dat overal in de tekst en tabellen er verwezen wordt naar duidelijke methodenamen (MP, ss, SSL zegt mij niets). Dus zoals in "method_combi" waar duidelijk aangegeven is welk type methode het is.